client/src/templates/Challenges/redux/create-question-epic.js

```diff
@@ -139,7 +142,7 @@ function createQuestionEpic(action$, state$, { window }) {
       const addCodeOne = i18next.t('forum-help.add-code-one');
```

Comment:
Hi there. Your implementation seems very nice so far. There is just one personal nitpick of mine. If it's not possible to grab the whole file, maybe we can grab just the edit region rather than force the camper to copy and paste the edit code? I can't see the edit region to any challenge being big enough to warrant things being simply to long. I don't believe campers appreciate having to post code when it doesn't seem necessary, and some have trouble keeping their formatting straight.

Reply:
> Hi there. Your implementation seems very nice so far. There is just one personal nitpick of mine. If it's not possible to grab the whole file, maybe we can grab just the edit region rather than force the camper to copy and paste the edit code? I can't see the edit region to any challenge being big enough to warrant things being simply to long. I don't believe campers appreciate having to post code when it doesn't seem necessary, and some have trouble keeping their formatting straight.

You are right. It is better showing only the editable region than nothing. I added a third fallback URI that sends only the editable region to the forum post even then if somehow that URI is bigger then it will fallback to the altURI where the user will have copy and paste the code.